nixos/modules/services/x11/desktop-managers/gnome3.nix

```
@@ -316,6 +316,8 @@ in
         gnome-shell
         gnome-shell-extensions
         gnome-themes-extra
+        nixos-artwork.wallpapers.simple-dark-gray
+        nixos-artwork.wallpapers.simple-dark-gray-bottom
```

```suggestion
        pkgs.nixos-artwork.wallpapers.simple-dark-gray
        pkgs.nixos-artwork.wallpapers.simple-dark-gray-bottom
```

as reported in https://github.com/NixOS/nixpkgs/pull/89881.

---

Will cherry-pick fa607bc93941ae1ccd2e3cd99a93feb214a2958e